version.sbt

```diff
@@ -1 +1 @@
- ThisBuild / version := "0.5.1"
```

Comment:
@realstraw looks like branch is pointing to an older master, need to pull the latest

Reply:
thanks, fixed